### DISCUSSION TITLE
Is there an easy way to handle resolution or window-size changes?

### DISCUSSION THREAD

**Discussion by visualcode-t** [bevyengine/bevy#1358] [Q&A] [answered]
When working with 2d Bevy, is there a way to use the camera to scale objects to the screen?

**Comment by mockersf** [comment, marked as answer]
I think you should look at the [`OrthographicCameraBundle`](https://github.com/bevyengine/bevy/blob/57f9ac18d7ac039d94466c479b70802597569259/crates/bevy_render/src/entity.rs#L83) added recently in #400

**Reply by nicolas-sabbatini** [reply]
I just spawn the camera or I need to do something else?
